### ISSUE TITLE
Weather APIs, sort of a mess?

### ISSUE THREAD

**Issue by rj45jack** [public-apis/public-apis#2579]
Hey, 

Forgive me in advance if this isn't something anyone should be worried about.

Under the Weather APIs there are two 'APIs' that are more like apps than APIs. One is [weather-api](https://github.com/robertoduessmann/weather-api). This 'API' is just a call to another App in the list. Which is [wttr](https://github.com/chubin/wttr.in). While wttr offers API calls they're just using [WorldWeatherOnline](https://www.worldweatheronline.com/) to action those calls.

In short. weather-api uses wttr which in-turn uses WorldWeatherOnline. I'm wondering where do we considering the line drawn between API and App (that uses someone else's API).

Thanks everyone.

**Comment by pawelborkar** [comment]
Thanks for pointing this out and opening this issue @rj45jack 

wttr is indeed an App and not an API so we need to remove it ASAP. Whereas weather-api does returns a JSON which contains weather information which can be used in the project, So I think we should keep it.

Further on this, would to like to open a PR regarding the removal of wttr from the list?

**Comment by rj45jack** [comment]
Thanks @pawelborkar. I'll start a PR shortly.
